lib/grpc/reflection/v1alpha/reflection.pb.ex

```diff
@@ -0,0 +1,586 @@
+defmodule Grpc.Reflection.V1alpha.ServerReflectionRequest do
+  @moduledoc false
+  use Protobuf, protoc_gen_elixir_version: "0.10.0", syntax: :proto3
+
+  def descriptor do
+    # credo:disable-for-next-line
```

Comment:
Was this autogenerated? We don't have credo in this project

Reply:
Don't worry like I said before I'm just bringing what makes sense in the Falco environment here first and then refactoring to what makes sense in elixir-grpc. This will come out in the final version.

Reply:
But I didn't look at it in depth and if this code is auto generated then the correct thing in my view would be not to mess with it since any build would bring back the comment regardless of having the creed or not. I will take a look and see what it is and what this comment is for more calmly when I resume PR